backend/corpora/common/entities/project.py

```diff
@@ -1,9 +1,58 @@
 from .entity import Entity
-from ..corpora_orm import DbProject
+from ..corpora_orm import DbProject, DbProjectLink
+from ..utils.uuid import generate_id
 
 
 class Project(Entity):
     table = DbProject
 
     def __init__(self, db_object: DbProject):
         super().__init__(db_object)
+
+    @classmethod
+    def create(
+        cls,
+        status: str,
+        name: str = "",
+        description: str = "",
+        owner: str = "",
+        s3_bucket: str = "",
+        tc_uri: str = "",
+        needs_attestation: bool = False,
+        processing_state: str = "",
+        validation_state: str = "",
+        links: list = None,
+    ) -> "Project":
+        """
+        Create a new Project and related objects and store in the database. UUIDs are generated for all new table
+        entries.
+
+        """
+        uuid = generate_id()
+
+        # Setting Defaults
+        links = links if links else []
+
+        #  Prevent accidentally linking an existing row to a different Project. This maintains the relationship of one
+        #  to many for links
+        [link.pop("id", None) for link in links]  # sanitize of ids
```

Comment:
Would remove the inline comment "# sanitize of ids". I actually have no idea what that means and the comment on the previous line makes it clear what this line is doing.

Reply:
```suggestion
        [link.pop("id", None) for link in links]
```